nengo/builder/operator.py

```diff
@@ -201,6 +201,42 @@ def step():
         return step
 
 
+class SlicedCopy(Operator):
+    """Copy from `a` to `b` with slicing: `b[b_slice] = a[a_slice]`"""
+    def __init__(self, a, b, a_slice=Ellipsis, b_slice=Ellipsis,
+                 kind='set', tag=None):
+        assert kind in ['set', 'inc', 'update']
```

Comment:
It doesn't look like `kind='update'` is ever used in this code? So I'd suggest either making a unit test for `update` (and changing the assert to an exception), or replacing this with a boolean for `set` versus `inc`.